pkg/kubelet/pod_workers.go

```diff
@@ -131,6 +142,7 @@ func (p *podWorkers) UpdatePod(pod *api.Pod, mirrorPod *api.Pod, updateComplete
 		// the channel is empty, so buffer of size 1 is enough.
 		podUpdates = make(chan workUpdate, 1)
 		p.podUpdates[uid] = podUpdates
+		updateType = SyncPodCreate
```

Comment:
I'm assuming that if you created a pod worker you're willing to believe the status in the pod without having generateStatus again, either because it is a brand new pod or the kubelet restarted. I'll add this as a comment.